pyproject.toml

```diff
@@ -36,6 +36,7 @@ dev = [
     "coverage[toml]",
     "flake8",
     "flake8-pyproject",
+    "freezegun",
```

Comment:
My preference would be to not introduce a new dependency for testing. Fetching order history is also time-based, but mocking out the responses and/or datetimes is sufficient. Could we rework your test to do the same to avoid this dependency?